```
Error running mint automation
+ mkdir -p 10799-9f339db/gopath/src/github.com/minio
+ git -C 10799-9f339db/gopath/src/github.com/minio clone --quiet https://github.com/minio/minio.git
+ git -C 10799-9f339db/gopath/src/github.com/minio/minio remote add minio https://github.com/minio/minio.git
+ git -C 10799-9f339db/gopath/src/github.com/minio/minio fetch --quiet minio pull/10799/head:pr10799
+ git -C 10799-9f339db/gopath/src/github.com/minio/minio checkout --quiet pr10799
+ GO111MODULE=on
+ GOPATH=/home/mint-auto/mint-auto/10799-9f339db/gopath
+ make -C 10799-9f339db/gopath/src/github.com/minio/minio --quiet
go: downloading github.com/minio/cli v1.22.0
go: downloading github.com/Azure/azure-storage-blob-go v0.10.0
go: downloading cloud.google.com/go v0.39.0
go: downloading github.com/minio/minio-go/v7 v7.0.6-0.20200929220449-755b5633803a
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading google.golang.org/api v0.5.0
go: downloading github.com/gorilla/mux v1.8.0
go: downloading github.com/willf/bloom v2.0.3+incompatible
go: downloading gopkg.in/jcmturner/gokrb5.v7 v7.3.0
go: downloading github.com/gorilla/rpc v1.2.0
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/djherbis/atime v1.0.0
go: downloading github.com/beevik/ntp v0.3.0
go: downloading github.com/colinmarc/hdfs/v2 v2.1.1
go: downloading github.com/prometheus/client_golang v1.0.0
go: downloading github.com/shirou/gopsutil v2.20.3-0.20200314133625-53cec6b37e6a+incompatible
go: downloading github.com/klauspost/reedsolomon v1.9.9
go: downloading github.com/cespare/xxhash/v2 v2.1.1
go: downloading github.com/willf/bitset v1.1.11
go: downloading github.com/google/uuid v1.1.1
go: downloading github.com/miekg/dns v1.1.8
go: downloading gopkg.in/yaml.v2 v2.2.8
go: downloading github.com/dchest/siphash v1.2.1
go: downloading github.com/klauspost/compress v1.10.3
go: downloading github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading github.com/dswarbrick/smart v0.0.0-20190505152634-909a45200d6d
go: downloading go.etcd.io/etcd/v3 v3.3.0-rc.0.0.20200707003333-58bb8ae09f8e
go: downloading github.com/elazarl/go-bindata-assetfs v1.0.0
go: downloading golang.org/x/sys v0.0.0-20200915084602-288bc346aa39
go: downloading golang.org/x/net v0.0.0-20200904194848-62affa334b73
go: downloading github.com/minio/md5-simd v1.1.0
go: downloading github.com/json-iterator/go v1.1.10
go: downloading github.com/montanaflynn/stats v0.5.0
go: downloading github.com/klauspost/readahead v1.3.1
go: downloading github.com/fatih/color v1.7.0
go: downloading github.com/prometheus/common v0.4.1
go: downloading github.com/prometheus/procfs v0.0.8
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.1
go: downloading github.com/modern-go/reflect2 v1.0.1
go: downloading github.com/tinylib/msgp v1.1.2
go: downloading github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: downloading google.golang.org/grpc v1.26.0
go: downloading golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a
go: downloading github.com/klauspost/cpuid v1.3.1
go: downloading github.com/minio/highwayhash v1.0.0
go: downloading github.com/go-sql-driver/mysql v1.5.0
go: downloading github.com/xdg/stringprep v1.0.0
go: downloading github.com/gogo/protobuf v1.3.1
go: downloading github.com/pkg/errors v0.9.1
go: downloading github.com/golang/protobuf v1.4.0
go: downloading github.com/ncw/directio v1.0.5
go: downloading github.com/eclipse/paho.mqtt.golang v1.2.0
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: downloading github.com/rs/xid v1.2.1
go: downloading google.golang.org/protobuf v1.22.0
go: downloading github.com/Shopify/sarama v1.24.1
go: downloading github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a
go: downloading github.com/eapache/queue v1.1.0
go: downloading golang.org/x/text v0.3.3
go: downloading github.com/rjeczalik/notify v0.9.2
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading github.com/gomodule/redigo v2.0.0+incompatible
go: downloading go.opencensus.io v0.22.3
go: downloading github.com/fatih/structs v1.1.0
go: downloading gopkg.in/ini.v1 v1.57.0
go: downloading go.uber.org/zap v1.14.1
go: downloading github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: downloading github.com/lib/pq v1.8.0
go: downloading github.com/gorilla/handlers v1.4.2
go: downloading github.com/pierrec/lz4 v2.4.0+incompatible
go: downloading golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421
go: downloading github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: downloading github.com/spaolacci/murmur3 v1.1.0
go: downloading github.com/rs/cors v1.7.0
go: downloading github.com/streadway/amqp v0.0.0-20190404075320-75d898a42a94
go: downloading github.com/philhofer/fwd v1.0.0
go: downloading github.com/nats-io/nats.go v1.10.0
go: downloading github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6
go: downloading github.com/nats-io/stan.go v0.7.0
go: downloading github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
go: downloading github.com/cheggaaa/pb v1.0.28
go: downloading github.com/Azure/azure-pipeline-go v0.2.2
go: downloading github.com/beorn7/perks v1.0.0
go: downloading github.com/nats-io/nkeys v0.1.4
go: downloading go.uber.org/atomic v1.6.0
go: downloading google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
go: downloading github.com/mattn/go-isatty v0.0.8
go: downloading gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: downloading github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: downloading github.com/eapache/go-resiliency v1.1.0
go: downloading go.uber.org/multierr v1.5.0
go: downloading github.com/nats-io/jwt v0.3.2
go: downloading github.com/hashicorp/go-uuid v1.0.1
go: downloading github.com/coredns/coredns v1.4.0
go: downloading gopkg.in/jcmturner/rpc.v1 v1.1.0
go: downloading github.com/hashicorp/vault/api v1.0.4
go: downloading github.com/mattn/go-runewidth v0.0.2
go: downloading gopkg.in/ldap.v3 v3.0.3
go: downloading github.com/secure-io/sio-go v0.3.0
go: downloading github.com/hashicorp/vault/sdk v0.1.13
go: downloading github.com/hashicorp/go-cleanhttp v0.5.1
go: downloading github.com/hashicorp/hcl v1.0.0
go: downloading github.com/mattn/go-ieproxy v0.0.1
go: downloading github.com/nats-io/nuid v1.0.1
go: downloading github.com/mitchellh/mapstructure v1.1.2
go: downloading gopkg.in/jcmturner/aescts.v1 v1.0.1
go: downloading github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: downloading github.com/klauspost/pgzip v1.2.1
go: downloading git.apache.org/thrift.git v0.13.0
go: downloading github.com/mattn/go-colorable v0.1.4
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/golang/snappy v0.0.1
go: downloading github.com/hashicorp/go-multierror v1.0.0
go: downloading github.com/googleapis/gax-go/v2 v2.0.4
go: downloading gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: downloading github.com/minio/selfupdate v0.3.1
go: downloading github.com/hashicorp/go-rootcerts v1.0.1
go: downloading golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: downloading github.com/hashicorp/go-retryablehttp v0.5.4
go: downloading github.com/minio/simdjson-go v0.1.5
go: downloading gopkg.in/square/go-jose.v2 v2.3.1
go: downloading github.com/alecthomas/participle v0.2.1
go: downloading github.com/coreos/go-semver v0.2.0
go: downloading github.com/hashicorp/go-sockaddr v1.0.2
go: downloading github.com/ryanuber/go-glob v1.0.0
go: downloading github.com/hashicorp/errwrap v1.0.0
go: downloading gopkg.in/olivere/elastic.v5 v5.0.86
go: downloading gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: downloading github.com/coreos/go-systemd/v22 v22.0.0
go: downloading github.com/nsqio/go-nsq v1.0.7
go: downloading github.com/tidwall/sjson v1.0.4
go: downloading github.com/tidwall/gjson v1.3.5
go: downloading github.com/minio/sio v0.2.1
go: downloading github.com/tidwall/pretty v1.0.0
go: downloading github.com/tidwall/match v1.0.1
go: downloading github.com/mailru/easyjson v0.7.1
# github.com/minio/minio/cmd
cmd/peer-rest-server.go:1057:66: undefined: peerRESTMethodReloadFormat
cmd/peer-rest-server.go:1057:126: server.ReloadFormatHandler undefined (type *peerRESTServer has no field or method ReloadFormatHandler)
make: *** [build] Error 2

```

--- a/cmd/erasure-server-sets.go
+++ b/cmd/erasure-server-sets.go
@@ -1264,17 +1264,6 @@ func (z *erasureServerSets) ListBuckets(ctx context.Context) (buckets []BucketIn
 	return buckets, nil
 }
 
-func (z *erasureServerSets) ReloadFormat(ctx context.Context, dryRun bool) error {
-	// No locks needed since reload happens in HealFormat under
-	// write lock across all nodes.
-	for _, zone := range z.serverSets {
-		if err := zone.ReloadFormat(ctx, dryRun); err != nil {
-			return err
-		}
-	}
-	return nil
-}
-
 func (z *erasureServerSets) HealFormat(ctx context.Context, dryRun bool) (madmin.HealResultItem, error) {
 	// Acquire lock on format.json
 	formatLock := z.NewNSLock(ctx, minioMetaBucket, formatConfigFile)
@@ -1304,15 +1293,6 @@ func (z *erasureServerSets) HealFormat(ctx context.Context, dryRun bool) (madmin
 		r.SetCount += result.SetCount
 		r.Before.Drives = append(r.Before.Drives, result.Before.Drives...)
 		r.After.Drives = append(r.After.Drives, result.After.Drives...)
-	}
-
-	// Healing succeeded notify the peers to reload format and re-initialize disks.
-	// We will not notify peers if healing is not required.
-	for _, nerr := range globalNotificationSys.ReloadFormat(dryRun) {
-		if nerr.Err != nil {
-			logger.GetReqInfo(ctx).SetTags("peerAddress", nerr.Host.String())
-			logger.LogIf(ctx, nerr.Err)
-		}
 	}
 
 	// No heal returned by all serverSets, return errNoHealRequired

--- a/cmd/erasure-sets.go
+++ b/cmd/erasure-sets.go
@@ -1141,81 +1141,6 @@ func formatsToDrivesInfo(endpoints Endpoints, formats []*formatErasureV3, sErrs 
 	return beforeDrives
 }
 
-// Reloads the format from the disk, usually called by a remote peer notifier while
-// healing in a distributed setup.
-func (s *erasureSets) ReloadFormat(ctx context.Context, dryRun bool) (err error) {
-	storageDisks, errs := initStorageDisksWithErrorsWithoutHealthCheck(s.endpoints)
-	for i, err := range errs {
-		if err != nil && err != errDiskNotFound {
-			return fmt.Errorf("Disk %s: %w", s.endpoints[i], err)
-		}
-	}
-	defer func(storageDisks []StorageAPI) {
-		if err != nil {
-			closeStorageDisks(storageDisks)
-		}
-	}(storageDisks)
-
-	formats, _ := loadFormatErasureAll(storageDisks, false)
-	if err = checkFormatErasureValues(formats, s.setDriveCount); err != nil {
-		return err
-	}
-
-	refFormat, err := getFormatErasureInQuorum(formats)
-	if err != nil {
-		return err
-	}
-
-	s.monitorContextCancel() // turn-off disk monitoring and replace format.
-
-	s.erasureDisksMu.Lock()
-
-	// Replace with new reference format.
-	s.format = refFormat
-
-	// Close all existing disks and reconnect all the disks.
-	for _, disk := range storageDisks {
-		if disk == nil {
-			continue
-		}
-
-		diskID, err := disk.GetDiskID()
-		if err != nil {
-			continue
-		}
-
-		m, n, err := findDiskIndexByDiskID(refFormat, diskID)
-		if err != nil {
-			continue
-		}
-
-		if s.erasureDisks[m][n] != nil {
-			s.erasureDisks[m][n].Close()
-		}
-
-		s.endpointStrings[m*s.setDriveCount+n] = disk.String()
-		if !disk.IsLocal() {
-			// Enable healthcheck disk for remote endpoint.
-			disk, err = newStorageAPI(disk.Endpoint())
-			if err != nil {
-				continue
-			}
-			disk.SetDiskID(diskID)
-		}
-
-		s.erasureDisks[m][n] = disk
-	}
-
-	s.erasureDisksMu.Unlock()
-
-	mctx, mctxCancel := context.WithCancel(GlobalContext)
-	s.monitorContextCancel = mctxCancel
-
-	go s.monitorAndConnectEndpoints(mctx, defaultMonitorConnectEndpointInterval)
-
-	return nil
-}
-
 // If it is a single node Erasure and all disks are root disks, it is most likely a test setup, else it is a production setup.
 // On a test setup we allow creation of format.json on root disks to help with dev/testing.
 func isTestSetup(infos []DiskInfo, errs []error) bool {
@@ -1335,13 +1260,8 @@ func (s *erasureSets) HealFormat(ctx context.Context, dryRun bool) (res madmin.H
 			}
 		}
 
-		// Save formats `format.json` across all disks.
-		if err = saveFormatErasureAllWithErrs(ctx, storageDisks, sErrs, tmpNewFormats); err != nil {
-			return madmin.HealResultItem{}, err
-		}
-
-		refFormat, err = getFormatErasureInQuorum(tmpNewFormats)
-		if err != nil {
+		// Save new formats `format.json` on unformatted disks.
+		if err = saveUnformattedFormat(ctx, storageDisks, tmpNewFormats); err != nil {
 			return madmin.HealResultItem{}, err
 		}
 
@@ -1349,21 +1269,12 @@ func (s *erasureSets) HealFormat(ctx context.Context, dryRun bool) (res madmin.H
 
 		s.erasureDisksMu.Lock()
 
-		// Replace with new reference format.
-		s.format = refFormat
-
-		// Disconnect/relinquish all existing disks, lockers and reconnect the disks, lockers.
-		for _, disk := range storageDisks {
-			if disk == nil {
+		for index, format := range tmpNewFormats {
+			if format == nil {
 				continue
 			}
 
-			diskID, err := disk.GetDiskID()
-			if err != nil {
-				continue
-			}
-
-			m, n, err := findDiskIndexByDiskID(refFormat, diskID)
+			m, n, err := findDiskIndexByDiskID(refFormat, format.Erasure.This)
 			if err != nil {
 				continue
 			}
@@ -1372,18 +1283,12 @@ func (s *erasureSets) HealFormat(ctx context.Context, dryRun bool) (res madmin.H
 				s.erasureDisks[m][n].Close()
 			}
 
-			s.endpointStrings[m*s.setDriveCount+n] = disk.String()
-			if !disk.IsLocal() {
-				// Enable healthcheck disk for remote endpoint.
-				disk, err = newStorageAPI(disk.Endpoint())
-				if err != nil {
-					continue
-				}
-				disk.SetDiskID(diskID)
-			}
-			s.erasureDisks[m][n] = disk
-
+			s.erasureDisks[m][n] = storageDisks[index]
+			s.endpointStrings[m*s.setDriveCount+n] = storageDisks[index].String()
 		}
+
+		// Replace with new reference format.
+		s.format = refFormat
 
 		s.erasureDisksMu.Unlock()
 

--- a/cmd/format-erasure.go
+++ b/cmd/format-erasure.go
@@ -704,28 +704,18 @@ func initErasureMetaVolumesInLocalDisks(storageDisks []StorageAPI, formats []*fo
 	return nil
 }
 
-// saveFormatErasureAllWithErrs - populates `format.json` on disks in its order.
+// saveUnformattedFormat - populates `format.json` on unformatted disks.
 // also adds `.healing.bin` on the disks which are being actively healed.
-func saveFormatErasureAllWithErrs(ctx context.Context, storageDisks []StorageAPI, fErrs []error, formats []*formatErasureV3) error {
-	g := errgroup.WithNErrs(len(storageDisks))
-
-	// Write `format.json` to all disks.
-	for index := range storageDisks {
-		index := index
-		g.Go(func() error {
-			if formats[index] == nil {
-				return errDiskNotFound
-			}
-			if errors.Is(fErrs[index], errUnformattedDisk) {
-				return saveFormatErasure(storageDisks[index], formats[index], true)
-			}
-			return nil
-		}, index)
+func saveUnformattedFormat(ctx context.Context, storageDisks []StorageAPI, formats []*formatErasureV3) error {
+	for index, format := range formats {
+		if format == nil {
+			continue
+		}
+		if err := saveFormatErasure(storageDisks[index], format, true); err != nil {
+			return err
+		}
 	}
-
-	writeQuorum := getWriteQuorum(len(storageDisks))
-	// Wait for the routines to finish.
-	return reduceWriteQuorumErrs(ctx, g.Wait(), nil, writeQuorum)
+	return nil
 }
 
 // saveFormatErasureAll - populates `format.json` on disks in its order.

--- a/cmd/fs-v1.go
+++ b/cmd/fs-v1.go
@@ -1536,12 +1536,6 @@ func (fs *FSObjects) DeleteObjectTags(ctx context.Context, bucket, object string
 	return fs.PutObjectTags(ctx, bucket, object, "", opts)
 }
 
-// ReloadFormat - no-op for fs, Valid only for Erasure.
-func (fs *FSObjects) ReloadFormat(ctx context.Context, dryRun bool) error {
-	logger.LogIf(ctx, NotImplemented{})
-	return NotImplemented{}
-}
-
 // HealFormat - no-op for fs, Valid only for Erasure.
 func (fs *FSObjects) HealFormat(ctx context.Context, dryRun bool) (madmin.HealResultItem, error) {
 	logger.LogIf(ctx, NotImplemented{})

--- a/cmd/gateway-unsupported.go
+++ b/cmd/gateway-unsupported.go
@@ -160,11 +160,6 @@ func (a GatewayUnsupported) DeleteBucketSSEConfig(ctx context.Context, bucket st
 	return NotImplemented{}
 }
 
-// ReloadFormat - Not implemented stub.
-func (a GatewayUnsupported) ReloadFormat(ctx context.Context, dryRun bool) error {
-	return NotImplemented{}
-}
-
 // HealFormat - Not implemented stub
 func (a GatewayUnsupported) HealFormat(ctx context.Context, dryRun bool) (madmin.HealResultItem, error) {
 	return madmin.HealResultItem{}, NotImplemented{}

--- a/cmd/notification.go
+++ b/cmd/notification.go
@@ -140,21 +140,6 @@ func (g *NotificationGroup) Go(ctx context.Context, f func() error, index int, a
 	}()
 }
 
-// ReloadFormat - calls ReloadFormat REST call on all peers.
-func (sys *NotificationSys) ReloadFormat(dryRun bool) []NotificationPeerErr {
-	ng := WithNPeers(len(sys.peerClients))
-	for idx, client := range sys.peerClients {
-		if client == nil {
-			continue
-		}
-		client := client
-		ng.Go(GlobalContext, func() error {
-			return client.ReloadFormat(dryRun)
-		}, idx, *client.host)
-	}
-	return ng.Wait()
-}
-
 // DeletePolicy - deletes policy across all peers.
 func (sys *NotificationSys) DeletePolicy(policyName string) []NotificationPeerErr {
 	ng := WithNPeers(len(sys.peerClients))

--- a/cmd/object-api-interface.go
+++ b/cmd/object-api-interface.go
@@ -114,7 +114,6 @@ type ObjectLayer interface {
 	CompleteMultipartUpload(ctx context.Context, bucket, object, uploadID string, uploadedParts []CompletePart, opts ObjectOptions) (objInfo ObjectInfo, err error)
 
 	// Healing operations.
-	ReloadFormat(ctx context.Context, dryRun bool) error
 	HealFormat(ctx context.Context, dryRun bool) (madmin.HealResultItem, error)
 	HealBucket(ctx context.Context, bucket string, dryRun, remove bool) (madmin.HealResultItem, error)
 	HealObject(ctx context.Context, bucket, object, versionID string, opts madmin.HealOpts) (madmin.HealResultItem, error)

--- a/cmd/peer-rest-client.go
+++ b/cmd/peer-rest-client.go
@@ -456,19 +456,6 @@ func (client *peerRESTClient) DeleteBucketMetadata(bucket string) error {
 	return nil
 }
 
-// ReloadFormat - reload format on the peer node.
-func (client *peerRESTClient) ReloadFormat(dryRun bool) error {
-	values := make(url.Values)
-	values.Set(peerRESTDryRun, strconv.FormatBool(dryRun))
-
-	respBody, err := client.call(peerRESTMethodReloadFormat, values, nil, -1)
-	if err != nil {
-		return err
-	}
-	defer http.DrainBody(respBody)
-	return nil
-}
-
 // cycleServerBloomFilter will cycle the bloom filter to start recording to index y if not already.
 // The response will contain a bloom filter starting at index x up to, but not including index y.
 // If y is 0, the response will not update y, but return the currently recorded information

--- a/cmd/peer-rest-common.go
+++ b/cmd/peer-rest-common.go
@@ -50,7 +50,6 @@ const (
 	peerRESTMethodLoadGroup              = "/loadgroup"
 	peerRESTMethodStartProfiling         = "/startprofiling"
 	peerRESTMethodDownloadProfilingData  = "/downloadprofilingdata"
-	peerRESTMethodReloadFormat           = "/reloadformat"
 	peerRESTMethodCycleBloom             = "/cyclebloom"
 	peerRESTMethodTrace                  = "/trace"
 	peerRESTMethodListen                 = "/listen"
@@ -72,7 +71,6 @@ const (
 	peerRESTIsGroup     = "is-group"
 	peerRESTSignal      = "signal"
 	peerRESTProfiler    = "profiler"
-	peerRESTDryRun      = "dry-run"
 	peerRESTTraceAll    = "all"
 	peerRESTTraceErr    = "err"
 

--- a/cmd/peer-rest-server.go
+++ b/cmd/peer-rest-server.go
@@ -578,45 +578,6 @@ func (s *peerRESTServer) LoadBucketMetadataHandler(w http.ResponseWriter, r *htt
 	}
 }
 
-// ReloadFormatHandler - Reload Format.
-func (s *peerRESTServer) ReloadFormatHandler(w http.ResponseWriter, r *http.Request) {
-	if !s.IsValid(w, r) {
-		s.writeErrorResponse(w, errors.New("Invalid request"))
-		return
-	}
-
-	vars := mux.Vars(r)
-	dryRunString := vars[peerRESTDryRun]
-	if dryRunString == "" {
-		s.writeErrorResponse(w, errors.New("dry-run parameter is missing"))
-		return
-	}
-
-	var dryRun bool
-	switch strings.ToLower(dryRunString) {
-	case "true":
-		dryRun = true
-	case "false":
-		dryRun = false
-	default:
-		s.writeErrorResponse(w, errInvalidArgument)
-		return
-	}
-
-	objAPI := newObjectLayerFn()
-	if objAPI == nil {
-		s.writeErrorResponse(w, errServerNotInitialized)
-		return
-	}
-
-	err := objAPI.ReloadFormat(GlobalContext, dryRun)
-	if err != nil {
-		s.writeErrorResponse(w, err)
-		return
-	}
-	w.(http.Flusher).Flush()
-}
-
 // CycleServerBloomFilterHandler cycles bloom filter on server.
 func (s *peerRESTServer) CycleServerBloomFilterHandler(w http.ResponseWriter, r *http.Request) {
 	if !s.IsValid(w, r) {
@@ -1093,7 +1054,6 @@ func registerPeerRESTHandlers(router *mux.Router) {
 
 	subrouter.Methods(http.MethodPost).Path(peerRESTVersionPrefix + peerRESTMethodStartProfiling).HandlerFunc(httpTraceAll(server.StartProfilingHandler)).Queries(restQueries(peerRESTProfiler)...)
 	subrouter.Methods(http.MethodPost).Path(peerRESTVersionPrefix + peerRESTMethodDownloadProfilingData).HandlerFunc(httpTraceHdrs(server.DownloadProfilingDataHandler))
-	subrouter.Methods(http.MethodPost).Path(peerRESTVersionPrefix + peerRESTMethodReloadFormat).HandlerFunc(httpTraceHdrs(server.ReloadFormatHandler)).Queries(restQueries(peerRESTDryRun)...)
 	subrouter.Methods(http.MethodPost).Path(peerRESTVersionPrefix + peerRESTMethodTrace).HandlerFunc(server.TraceHandler)
 	subrouter.Methods(http.MethodPost).Path(peerRESTVersionPrefix + peerRESTMethodListen).HandlerFunc(httpTraceHdrs(server.ListenHandler))
 	subrouter.Methods(http.MethodPost).Path(peerRESTVersionPrefix + peerRESTMethodBackgroundHealStatus).HandlerFunc(server.BackgroundHealStatusHandler)

